integration/tests/main_test.go

```diff
@@ -23,6 +23,7 @@ func TestMain(m *testing.M) {
 	appURL := os.Getenv("AIA_APP_URL")
 	config.BasePath = appURL
 	_, config.Debug = os.LookupEnv("AIA_INTEGRATION_DEBUG")
+	config.Debug = true
```

Comment:
This seems to completely remove any flakes :(

Reply:
I assume forcing I/O after every request prevents one or more race conditions from happening. Let's see if this is true...